.gitignore

```diff
@@ -6,3 +6,5 @@ node_modules
 *.DS_Store
 logs
 *.log
+deviceconfig.js
+*.vscode
```

Comment:
Actually, maybe a better name would be testconfig.json ?

---
In reply to: [73765456](https://github.com/openT2T/translators/pull/37#discussion_r73765456) [](ancestors = 73765456)